packages/neo-one-website/src/components/common/Sidebar.tsx

```diff
@@ -61,14 +63,28 @@ export const Sidebar = ({ sections, tutorial, ...props }: Props) => (
     <DesktopStyledBox>
       <StyledList>
         {tutorial ? <SidebarHeader /> : undefined}
-        {sections.map(
-          ({ section, subsections }) =>
-            tutorial ? (
-              <TutorialSection section={section} subsections={subsections} />
-            ) : (
-              <DocSection section={section} subsections={subsections} />
-            ),
-        )}
+        <ActiveSectionContainer>
```

Comment:
This should wrap just the `TutorialSection` since it's not used at all by `DocSection`.